JWBestPracticeApps/FeedTableViewController/FeedTableViewController/Feed/Playlist.swift

```diff
@@ -0,0 +1,94 @@
+//
+//  Playlist.swift
+//  FeedTableViewController
+//
+//  Created by Amitai Blickstein on 6/26/22.
+//
+
+import Foundation
+import UIKit
+
+/// Setting to `false` will load the landscape videos.
+private let showPortraitVideos = true
+
+struct Playlist {
+    
+    /// The "BPA Manual, landscape" playlist, id: [d7sk9orq](https://cdn.jwplayer.com/v2/playlists/d7sk9orq)
```

Comment:
I'm not sure what the purpose of these two commented out lines are?

Reply:
Removed, as above.